src/issues/mod.rs

```diff
@@ -490,6 +490,7 @@ pub struct Issue {
     pub assignee: Option<User>,
     pub locked: bool,
     pub comments: u64,
+    pub pull_request: Option<serde_json::Value>,
```

Comment:
I'm looking at the example payload [here](https://developer.github.com/v3/issues/#response-2) which lists the following fields.  any reason why we wouldn't want to parse this out as a deserializable struct with well known fields?

```json
  "pull_request": {
    "url": "https://api.github.com/repos/octocat/Hello-World/pulls/1347",
    "html_url": "https://github.com/octocat/Hello-World/pull/1347",
    "diff_url": "https://github.com/octocat/Hello-World/pull/1347.diff",
    "patch_url": "https://github.com/octocat/Hello-World/pull/1347.patch"
  },
```

Reply:
That's a good point. I'll go ahead and make that change tomorrow.

I was thinking that this could be used to make a request for the full pull request information in the future if we come up with a clean way of doing that.

Reply:
I have amended my commit with this change.